helpdesk/email.py

```diff
@@ -523,7 +525,19 @@ def object_from_message(message, queue, logger):
                 if part['Content-Transfer-Encoding'] == '8bit' and part.get_content_charset() == 'utf-8':
                     body = body.decode('unicode_escape')
                 body = decodeUnknown(part.get_content_charset(), body)
-                body = EmailReplyParser.parse_reply(body)
+                # have to use django_settings here so overwritting it works in tests
+                # the default value is False anyway
+                if ticket is None and getattr(django_settings, 'HELPDESK_FULL_FIRST_MESSAGE_FROM_EMAIL', False):
```

Comment:
Does this need to be a setting? Can it not be on by default? If people find the forwarded parts of the messages are too long, we can add javascript later to hide them.

Reply:
the same; it uglifies the first message very much, not sure if everyone is going to be happy about it. If no backwards compatibility is issue then making this default is cool.